avro/swagger/src/test/resources/expected-happy-model.json

```diff
@@ -1,135 +1,116 @@
-[
-  {
-    "id": "HappyModel",
-    "description": "This model can be used to test most test cases.\nThis comment can be used to test multiline descriptions\nbecause it's a multiline comment.",
-    "required": [
-      "directly_nested_model",
-      "enum_field",
-      "model_array_field",
-      "primitive_array_field",
-      "string_field",
-      "bytes_field",
-      "int_field",
-      "long_field",
-      "float_field",
-      "double_field",
-      "boolean_field"
-    ],
-    "properties": {
-      "directly_nested_model": {
-        "$ref": "NestedHappyModel",
-        "description": "Make sure the field comment, not the record comment, is used for the field.\nAnd make sure multiline field comments work right, too."
-      },
-      "optional_nested_model": {
-        "$ref": "NestedHappyModel",
-        "description": "Comments work on optional fields."
-      },
-      "enum_field": {
-        "type": "string",
-        "description": "Comments work on enum fields.",
-        "enum": [
-          "EBULLIENT",
-          "BUOYANT",
-          "PEPPY",
-          "FULL_OF_BEANS"
-        ]
-      },
-      "optional_enum_field": {
-        "type": "string",
-        "enum": [
-          "EBULLIENT",
-          "BUOYANT",
-          "PEPPY",
-          "FULL_OF_BEANS"
-        ]
-      },
-      "model_array_field": {
-        "type": "array",
-        "items": {
-          "$ref": "NestedHappyModel"
-        }
-      },
-      "optional_model_array_field": {
-        "type": "array",
-        "items": {
-          "$ref": "NestedHappyModel"
+{
+  "swagger" : "2.0",
+  "definitions" : {
+    "HappyModel" : {
+      "type" : "object",
+      "required" : [ "boolean_field", "bytes_field", "directly_nested_model", "double_field", "enum_field", "float_field", "int_field", "long_field", "model_array_field", "optional_model_array_field", "optional_primitive_array_field", "primitive_array_field", "string_field" ],
```

Comment:
Why are these optional fields in the required section?

Reply:
aaa0b512 oops, all arrays were being marked required.